package.json

```diff
@@ -27,7 +27,12 @@
     "test:dev": "run-s test:dev:*",
     "test:ci": "run-s test:ci:*",
     "test:dev:ava": "ava",
-    "test:ci:ava": "nyc -r lcovonly -r text -r json ava"
+    "test:ci:ava": "nyc -r lcovonly -r text -r json ava",
+    "prepublishOnly": "npx npm-run-all -s prepublishOnly:*",
```

Comment:
Use `npx` in case `npm-run-all` hasn't been installed yet

Reply:
`npm publish` should not be run if `npm install` has not been run yet, so it would be proper to fail?
Please let me know if I missed something.

Reply:
We run `npm ci` as a part of the pre-publish script to sync the dependencies (after doing the `git pull`) so the publish should succeed.
I'm good with dropping `npx` as it is quite an edge case (someone publishing without a working setup).

Reply:
Yes, this does look like an edge case. I would expect most of us to have run `npm install` in the repository before trying to publish it.

Reply:
Fixed in https://github.com/netlify/eslint-config-node/pull/89/commits/684e58525ee7c33f50ad319914af1cb179d0e5d9